app/uk/gov/hmrc/economiccrimelevyregistration/viewmodels/checkAnswers/CheckYourAnswersViewModel.scala

```diff
@@ -214,16 +214,16 @@ case class CheckYourAnswersViewModel(
           hasSecondContactDetailsPresentChanged,
           SecondContactSummary.row(registration.contacts.secondContact)
         ) ++ addIf(
-          hasSecondContactNameChanged,
+          hasSecondContactNameChanged && hasSecondContact,
```

Comment:
Make this it's own appropriately named viewModel property/val

Reply:
Should this be in TrackRegistrationChanges? As it's not necessarily a change to the registration, it's just confirming if the secondContact is present